cmd/kube-proxy/app/server.go

```diff
@@ -306,6 +311,20 @@ func (o *Options) applyDeprecatedHealthzPortToConfig() {
 	o.config.HealthzBindAddress = fmt.Sprintf("%s:%d", o.config.HealthzBindAddress, o.healthzPort)
 }
 
+func (o *Options) applyDeprecatedMetricsPortToConfig() {
+	if o.metricsPort == 0 {
```

Comment:
I don't understand this.  It's a new flag, so the default will be 0.  In that case we nuke the user's requested MetricsBindAddress ?  Or is the default not 0?

Reply:
The default is set to `ports.ProxyStatusPort`, which should be the same as before (10249).

Reply:
where is that defaulted?

Reply:
I see it now.  Thanks!